packages/datetime/test/datePickerTests.tsx

```diff
@@ -26,6 +30,139 @@ describe("<DatePicker>", () => {
         assert.isUndefined(root.state("selectedDay"));
     });
 
+    describe("reconciliates dayPickerProps", () => {
+        it("week starts with firstDayOfWeek value", () => {
+            const selectedFirstDay = 3;
+            const wrapper = mount(<DatePicker dayPickerProps={{ firstDayOfWeek: selectedFirstDay }} />);
+            const firstWeekday = wrapper.find("Weekday").first();
+            assert.equal(firstWeekday.prop("weekday"), selectedFirstDay);
+        });
+
+        it("doesnt show outside days if specified", () => {
+            const defaultValue = new Date(2017, Months.SEPTEMBER, 1, 12);
+            const wrapper = mount(
+                <DatePicker defaultValue={defaultValue} dayPickerProps={{ enableOutsideDays: false }} />,
+            );
+
+            const dayIsHidden = (day: any): boolean =>
```

Comment:
Put the helping verb first: `isDayHidden`. Also, let's add minimal type-checking: I think `ReactWrapper<any, any>` is the expected type. This requires:

```tsx
import { ReactWrapper } from "enzyme";
```